crates/rpc/src/types/query.rs

```diff
@@ -4,14 +4,18 @@ use serde::{Deserialize, Serialize};
 use super::id::ValidatorID;
 
 #[derive(Debug, Serialize, Deserialize)]
-pub struct RandaoQuery {
+pub struct EpochQuery {
     pub epoch: Option<u64>,
 }
 
 #[derive(Debug, Deserialize)]
 pub struct SlotQuery {
     pub slot: Option<u64>,
 }
+#[derive(Debug, Deserialize)]
```

Comment:
```suggestion
}

#[derive(Debug, Deserialize)]
```

please review your PR before you submit it, that there is a new line between functions/objects. I need to point this out quite often